docs/Local_Login.md

```diff
@@ -0,0 +1,15 @@
+# How To Get Login Working Locally
+
+## Step-by-step Guide
+
+1. Run the shell script, `generate_ssl_certs.sh` located in the login folder.
```

Comment:
This has moved to `tools/generate-ssl-certs.sh` and also currently only works on Linux/macOS (worth mentioning for Windows users).

Reply:
Agreed with @humphd .

Reply:
Actually, this got missed.  Still need to update this.